backend/blueprint/api/controllers/assetHelper.py

```diff
@@ -26,4 +26,4 @@ def getDefaultBackgrounds():
             }
             resp.append(obj)
     resp_ = {'data': resp}
-    return jsonify(resp_)
+    return jsonify(Response(200).generateMessage(resp_, 'Image Response'))
```

Comment:
This won't work as ember relies on json api convention data or rest data (for which each key should have a different model). Please go through the ember docs at least once.

Reply:
This PR strictly follows http://jsonapi.org/format/